server/rooms.ts

```diff
@@ -1368,6 +1368,12 @@ export class BasicChatRoom extends BasicRoom {
 		Rooms.rooms.delete(oldID);
 		Rooms.rooms.set(newID, this as ChatRoom);
 
+		for (const [alias, roomid] of Rooms.aliases.entries()) {
+			if (roomid === oldID) {
+				Rooms.aliases.set(alias, newID);
```

Comment:
Unrelatedly, do we set an alias from the old room to the new room? That should probably be done.

Reply:
Sure.